SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -593,6 +594,11 @@ void ActorSystem::EntityAdded(const Worker_EntityId EntityId)
 	{
 		AuthorityGained(EntityId, AuthoritativeComponentSet);
 	}
+
+	if (!NetDriver->GetWorld()->GetGameInstance()->IsDedicatedServerInstance())
```

Comment:
We want to perform these checks before the actor is exposed to the actor system, otherwise we'll get state popping on actors already instantiated